cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -67,7 +68,7 @@
     @Test
     public void testGenerateDeployment() {
 
-        Deployment dep = entityOperator.generateDeployment(true);
+        Deployment dep = entityOperator.generateDeployment(Collections.EMPTY_MAP, true);
```

Comment:
The `emptyMap()` method is preferred (it doesn't matter here, but `EMPTY_MAP` can become a bad habit, `emptyMap()` allows the compiler to infer the type argument).